src/app/machines/components/MachineHeaderForms/MachineActionFormWrapper/CloneForm/CloneFormFields/CloneFormFields.test.tsx

```diff
@@ -117,35 +111,30 @@ describe("CloneFormFields", () => {
   });
 
   it("applies different styling depending on clone selection state", async () => {
-    const machine = machineFactory({ system_id: "abc123" });
+    const machine = machineDetailsFactory({ system_id: "abc123" });
     state.machine.items = [machine];
-    const store = mockStore(state);
-    const wrapper = mount(
-      <Provider store={store}>
-        <Formik
-          initialValues={{ interfaces: false, source: "", storage: false }}
-          onSubmit={jest.fn()}
-        >
-          <CloneFormFields
-            selectedMachine={null}
-            setSelectedMachine={jest.fn()}
-          />
-        </Formik>
-      </Provider>
+    renderWithMockStore(
+      <Formik
+        initialValues={{ interfaces: false, source: "", storage: false }}
+        onSubmit={jest.fn()}
+      >
+        <CloneFormFields
+          selectedMachine={machine}
+          setSelectedMachine={jest.fn()}
+        />
+      </Formik>,
+      { state }
     );
-    const getTableClass = () =>
-      wrapper.find("MainTable.clone-table--network").prop("className");
+    let table = screen.getAllByRole("grid")[0];
```

Comment:
Provided you add an aria-label to `CloneNetworkTable`:

```suggestion
    let table = screen.getByRole("grid", { name: "Clone network" });
```